android/ouinet/build.gradle

```diff
@@ -63,3 +63,11 @@ dependencies {
     testImplementation 'com.android.support.test:runner:1.0.2'
     testImplementation 'org.assertj:assertj-core:3.11.1'
 }
+
+ext {
+    PUBLISH_GROUP_ID = 'ie.equalit.ouinet'
+    PUBLISH_VERSION = '1.5.0'
```

Comment:
Just a reminder that this version may be extracted from `version.txt` in the main source directory.

Reply:
Hi @ivilata, the change is implemented now in 4b3c78f0